docs/streaming-kafka-integration.md

```diff
@@ -181,7 +181,20 @@ Next, we discuss how to use this approach in your streaming application.
 		);
 	</div>
 	<div data-lang="python" markdown="1">
-		Not supported yet
+		offsetRanges = []
+
+		def storeOffsetRanges(rdd):
+		    del offsetRanges[:]
+		    offsetRanges.extend(rdd.offsetRanges())
```

Comment:
I tried that and couldn't get it working - seems to be related to a new object being created instead of updating the old one.

However, if I add `global offsetRanges` before, that seems to work fine. I'll push that change out.